content/swiftui/concepts/views/terms/rectangle/rectangle.md

```diff
@@ -0,0 +1,48 @@
+---
+Title: 'Rectangle'
+Description: 'A rectangular shape aligned inside the frame of the view containing it.'
+Subjects:
+  - 'Mobile Development'
+  - 'Computer Science'
+Tags:
+  - 'Rectangle'
+  - 'SwiftUI'
+  - 'SwiftUI Views'
+  - 'Views'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+The **`rectangle`** A rectangular shape aligned inside the frame of the view containing it.
+
+## Syntax
+
+```pseudo
+var body: some View {
+  Rectangle()
+}
+```
+
+SwiftUI has several built-in shapes, such as `Rectangle`, circles, and capsules, and you can create, color, and place each shape as needed.
+
+## Example
+
+You can change it to the desired shape by attaching a modifier to the Rectangle.
+
+```swift
+var body: some View {
+    Rectangle()
+        .fill(Color.green)
+        .frame(width: 100, height: 100)
+        .padding(20)
+        .background(Color.yellow)
+        .cornerRadius(10.0)
+}
+```
+
+Order matters here; There is a yellow rectangle with a border's width of 20 and a rounded corner of 10. And there's a green rectangle that's 100 in length and 100 in width.
```

Comment:
```suggestion
Order matters here; there is a yellow rectangle with a border width of `20` and a rounded corner of `10.0`. Additionally, there's a green rectangle that's `100` in length and `100` in width.
```

Reply:
Can we explicitly define the unit of measurement for all these numbers here?

Reply:
I referred to the units in the official document. There was no point for padding and no unit for corner radius. Do we need a unit for corner radius? If necessary, could you tell me which unit to use?

https://developer.apple.com/documentation/charts/chart/padding(_:_:)
https://developer.apple.com/documentation/swiftui/view/cornerradius(_:antialiased:)

Reply:
@rriver2 yeah, can we use the term "points." This is a unit that's specific to Apple it seems.

Reply:
I changed it !